cmd/lora-gateway-bridge/cmd/configfile.go

```diff
@@ -117,6 +117,26 @@ clean_session={{ .Backend.MQTT.CleanSession }}
 # a random id will be generated. This requires clean_session=true.
 client_id="{{ .Backend.MQTT.ClientID }}"
 
+# Last Will topic
+#
+# Set Last Will topic, where Last Will message will be published
+last_will_topic="{{ .Backend.MQTT.LastWillTopic }}"
```

Comment:
Could you move this under the `config_topic_template` and call this `last_will_topic_template`? Thus:

```
last_will_topic_template="{{ .Backend.MQTT.LastWillTopicTemplate }}"
```